nionswift_plugin/usim/ScanDevice.py

```diff
@@ -77,8 +77,11 @@ def __init__(self) -> None:
         self.__scan_shape_pixels = Geometry.IntSize()
         self.__pixel_size_nm = Geometry.FloatSize()
         self.flyback_pixels = 2
+        self.flyback_time_us = 300
+        self.pixel_time_us = 10
+        self.__line_start_time = 0.0
         self.__n_flyback_pixels = 0
-        self.__current_line = 0
+        self.__current_line = -1
```

Comment:
As a general comment, I'm not in favor of making -1 a special value. It forces signed comparisons which can have unforeseen issues down the road. I didn't follow through the logic closely, though.